Program.cs

```diff
@@ -54,36 +55,50 @@ public static async Task<int> Main(string[] args)
                     await MigrateAssetsAsync(context, assetOptions, context.GetCancellationToken());
                 });
 
-// disable storage migrate option until ready
-/*
-            var storageOptionsBinder = new StorageOptionsBinder();
-            var storageCommand = storageOptionsBinder.GetCommand("storage", @"Directly migrate the assets from the storage account.
-Doesn't require the Azure media services to be running.
+
+            var cleanupOptionsBinder = new CleanupOptionsBinder();
+            var cleanupCommand = cleanupOptionsBinder.GetCommand("cleanup", @"Do the cleanup of AMS account or Storage account
 Examples:
-amsmigrate storage -s <subscription id> -g <resource group> -n <source storage account> -o <output storage account> -t path-template
-");
-            rootCommand.Add(storageCommand);
-            storageCommand.SetHandler(async context =>
-            {
-                var globalOptions = globalOptionsBinder.GetValue(context.BindingContext);
-                var storageOptions = storageOptionsBinder.GetValue(context.BindingContext);
-                await MigrateStorageAsync(globalOptions, storageOptions, context.GetCancellationToken());
-            });
-*/
-
-// disable key migrate option until ready
-/*
-            var keyOptionsBinder = new KeyOptionsBinder();
-            var keysCommand = keyOptionsBinder.GetCommand();
-            rootCommand.Add(keysCommand);
-            keysCommand.SetHandler(
+cleanup -s <subscriptionid> -g <resourcegroup> -n <account> -ax true                                                                                 
```

Comment:
The main goal of this command is to remove the selected input assets,  the cleanup of the ams account is just a bonus option.  It is better to put two examples, one for assets cleanup, the other for ams account cleanup.